src/aggregator/aggregator/aggregator.go

```diff
@@ -342,21 +384,6 @@ func (agg *aggregator) Close() error {
 	return nil
 }
 
-func (agg *aggregator) passWriter() (writer.Writer, error) {
-	agg.RLock()
-	defer agg.RUnlock()
-
-	if agg.state != aggregatorOpen {
-		return nil, errAggregatorNotOpenOrClosed
-	}
-
-	if agg.electionManager.ElectionState() == FollowerState {
-		return writer.NewBlackholeWriter(), nil
-	}
```

Comment:
should be fine to remove this 2nd check, as double-writing metrics in this rare case is not the end of the world.